icons/clock-arrow-up.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M13.228 21.925A10 10 0 1 1 21.994 12.338" />
```

Comment:
Closed gap.
```suggestion
  <path d="M14 21.798a10 10 0 1 1 7.994-9.46" />
```
<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M12+6v6l1.562.781%22+/%3E%3Cpath+d=%22m14+18+4-4+4+4%22+/%3E%3Cpath+d=%22M14+21.798a10+10+0+1+1+7.994-9.46%22+/%3E%3Cpath+d=%22M18+22v-8%22+/%3E"><img alt="icons" width="120" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTIgNnY2bDEuNTYyLjc4MSIgLz4KICA8cGF0aCBkPSJtMTQgMTggNC00IDQgNCIgLz4KICA8cGF0aCBkPSJNMTQgMjEuNzk4YTEwIDEwIDAgMSAxIDcuOTk0LTkuNDYiIC8+CiAgPHBhdGggZD0iTTE4IDIydi04IiAvPgo8L3N2Zz4=.svg"/><br/>Open lucide studio</a>

Reply:
No, actually, you didn't. :)

Mine vs. yours:
<img width="914" alt="image" src="https://github.com/lucide-icons/lucide/assets/17746067/d719218c-c355-4dc3-9bcb-8a9a67e9a49b">

Reply:
perfection!